app/views/partners/PagePartners/index.vue

```diff
@@ -0,0 +1,177 @@
+<template lang="pug">
+.container-fluid#partners-page
+  .width-container.text-center.row
+    .col-lg-12
+      h1.text-h1.text-navy(data-i18n="partners.headline")
+  .width-container.text-center.row
+    .col-lg-12
+      a.btn.btn-primary.btn-lg(href="https://form.typeform.com/to/loZIh0I9" target="_blank") {{ $t('general.contact_us') }}
+  .width-container.text-center.row
+    .col-lg-12
+      h5.text-h5.text-teal(data-i18n="partners.below_headline_blurb")
+
+  .width-container.row.justify-content-center
+    .col-sm-offset-1.col-sm-6.col-xs-12
+      p.text-navy
+        strong(data-i18n="partners.coding_title")
+        | &nbsp;
+        span(data-i18n="partners.coding_blurb")
+    .col-sm-3.col-xs-8.col-xs-offset-2.col-sm-offset-0
+      img.img-responsive(src="/images/pages/partners/coding.jpg")
+
+  .width-container.row.justify-content-center
+    .col-sm-offset-1.col-sm-3.col-xs-8.col-xs-offset-2
+      img.img-responsive(src="/images/pages/partners/powered-by-hyper.png")
+    .col-sm-6
+      p.text-navy
+        strong(data-i18n="partners.esports_title")
+        | &nbsp;
+        span(data-i18n="[html]partners.esports_blurb" :data-i18n-options="`${JSON.stringify(i18nData)}`")
+
+  .width-container.row
+    .col-lg-12
+      p.text-navy
+        strong(data-i18n="partners.distribution_title")
+        | &nbsp;
+        span(data-i18n="[html]partners.distribution_blurb" :data-i18n-options="`${JSON.stringify(i18nData)}`")
```

Comment:
most of them don't need all i18nOptions so could have just passed needed data directly here too. But this works too!